PresentScreenings/Entities/Screening.cs

```diff
@@ -428,13 +428,24 @@ public string ShortFriendsString(bool includeMe = false)
             return builder.ToString();
         }
 
-        public string ScreeningTitleIfDifferent()
+        public string ScreeningTitleIfDifferent(int? filmId=null)
         {
-            if (ScreeningTitle == FilmTitle)
+            string title = string.Empty;
+            if (filmId != null)
             {
-                return string.Empty;
+                if (filmId != FilmId)
+                {
+                    title = FilmTitle;
+                }
+            }
+            if (title == string.Empty)
+            {
+                if (ScreeningTitle != FilmTitle)
+                {
+                    title = ScreeningTitle;
+                }
             }
-            return string.Format($" - {ScreeningTitle}");
+            return title == string.Empty ? title : $" - {title}";
         }
```

Comment:
Good work!
Allowing combination titles to stand out against regular screenings.

Reply:
☺️